draft-ietf-tls-subcerts.md

```diff
@@ -220,19 +220,23 @@ This document defines the following extension code point.
 ~~~~~~~~~~
 
 A client which supports this document SHALL send an empty
-"delegated_credential" extension in its ClientHello.
+"delegated_credential" extension in its ClientHello.  If the client receives a
+delegated credential without indicating support, then the client SHOULD abort with
+an "unexpected_message" alert.
 
-If the extension is present, the server MAY send a DelegatedCredential
+If the extension is present, the server MAY send a delegated credential
 extension.  If the extension is not present, the server MUST NOT send a
 credential.  A credential MUST NOT be provided unless a Certificate message is
-also sent.
+also sent.  The server MUST ignore the extension unless TLS 1.2 or later is
+negotiated.
 
-When negotiating TLS 1.3, and using Delegated credentials, the server MUST send
-the DelegatedCredential as an extension in the CertificateEntry of its
-end-entity certificate.  When negotiating TLS 1.2, the DelegatedCredential MUST
-be sent as an extension in the ServerHello.
+When negotiating TLS 1.3, and using delegated credentials, the server MUST send
+the delegated credential as an extension in the CertificateEntry of its
+end-entity certificate; the client SHOULD ignore delegated credentials sent as
```

Comment:
What about an error if it is sent in any other certificate

Reply:
That's possibility, but it just doesn't seem necessary to me.